app/models/application_plan.rb

```diff
@@ -17,7 +17,6 @@ class ApplicationPlan < Plan
       where(services: { account_id: provider.id }).joins(:service).references(:service).readonly(false)
     end
   }
-
```

Comment:
??? 👀

Reply:
Since all the validations are in plan.rb, Gui suggested to add in the same file and not in application_plan.rb
Hence the change. Thoughts?

Reply:
I agree with Gui, what does not imply removing the empty space between these 2 methods/scopes :stuck_out_tongue_closed_eyes:

Reply:
Done